rio_cogeo/cogeo.py

```diff
@@ -70,6 +70,7 @@ def cog_translate(  # noqa: C901
     temporary_compression: str = "DEFLATE",
     colormap: Optional[Dict] = None,
     additional_cog_metadata: Optional[Dict] = None,
+    gdal_cog: bool = False,
```

Comment:
Seems like a good name could be `use_gdal` or similar?

Reply:
Hopefully that name is a little more clear, but maybe it could also be confusing because whether or not you use the COG driver in GDAL 3.1+, you use GDAL 🤷‍♂️

Reply:
so `use-gdal-cog` ? 😭

Reply:
Or maybe `--use-cog-driver`?  It's using GDAL regardless, this just determines which driver is used (GTiff vs COG).  Could also just expose a `--driver` flag which defaults to GTiff but may be set to COG.

Reply:
I like `--use-cog-driver` better 🙏